packages/cojson/src/coValues/group.ts

```diff
@@ -558,13 +719,20 @@ function isMorePermissiveAndShouldInherit(
   }
 
   if (roleInParent === "writer") {
-    return !roleInChild || roleInChild === "reader";
+    return (
+      !roleInChild || roleInChild === "reader" || roleInChild === "writeOnly"
+    );
   }
 
   if (roleInParent === "reader") {
     return !roleInChild;
   }
```

Comment:
Configured the `reader` role to not be inherited by writerOnly members.

Considering them incompatible and kind-of at the same level.

Reply:
correct